Sources/SecureXPC/Server/XPCServiceServer.swift

```diff
@@ -40,9 +40,20 @@ internal class XPCServiceServer: XPCServer {
 
         return (uint32ToString(packageType.bigEndian), uint32ToString(packageCreator.bigEndian))
     }
-    
+
+    private lazy var xpcServiceName: String = {
+        guard let bundleID = Bundle.main.bundleIdentifier else {
+            fatalError("XPC Services need to have a bundle identifier set.")
```

Comment:
Hmm, instead of taking this fatalError approach, what if it was determined during the `_forThisXPCService()` function and if it's not present it throws something like `XPCError.misconfiguredXPCService`? That'd make it more consistent with behavior for the already thrown `XPCError.notXPCService` error.

Reply:
Great idea.